jupyter_book/commands/__init__.py

```diff
@@ -225,6 +225,36 @@ def toc(path, filename_split_char, skip_text, output_folder):
     _message_box(f"Table of Contents written to {output_file}")
 
 
+@main.command()
+@click.argument("path-output")
```

Comment:
maybe this should be `path-build` instead of `path-output` so that it's clear that this will only work on the `_build` folder

Reply:
ohhh nevermind, I see that this is actually the directory *where the output folder is*. That makes sense then, I think we could just go with `path-output`

Reply:
alternatively what if we did `path-book` here?

Reply:
@najuzilu I think this is the only remaining thing to resolve - if you think `path-outputs` is better that's fine too, just lemme know what you think

Reply:
I prefer `path-book`. I'll fix this and rebase so the branch reflects the latest master.

Reply:
@choldgraf  can I also fix this under `create` command? I think that's what I followed when I named this.

Reply:
yep, +1 to changing in create as well